app/views/content_items/guide_single.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :extra_head_content do %>
+  <%= machine_readable_metadata(
+    schema: :article,
+    canonical_url: @content_item.canonical_url,
+    body: @content_item.current_part_body
+  ) %>
+<% end %>
+
+<% if MachineReadable::YamlFaqPageSchemaPresenter.configured?(@content_item) %>
+<script type="application/ld+json">
+  <%= raw MachineReadable::YamlFaqPageSchemaPresenter.new(@content_item).structured_data.to_json %>
+</script>
+<% end %>
+
+<% content_for :simple_header, true %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', { title: @content_item.content_title } %>
+
+    <% if @content_item.show_guide_navigation? %>
+      <aside class="part-navigation-container" role="complementary">
+        <%= render "govuk_publishing_components/components/contents_list", aria_label: 'Pages in this guide', contents: @content_item.part_link_elements, underline_links: true %>
+      </aside>
+    <% end %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <% if @content_item.has_parts? %>
+
+      <% @content_item.parts.each_with_index do |part, index| %>
+      <section class="govuk-!-padding-bottom-9" aria-labelledby="<%= part["slug"] %>">
+        <%= render "govuk_publishing_components/components/heading", {
+            text: part["title"],
+            id: part["slug"],
+            heading_level: 1,
```

Comment:
This would need to be a level 2 heading – Although this means the content we pass trough govspeak on L42 would need to have the heading levels moved down (H2->H3). While the heading change won't affect the page visually, the content change (coming from `part["body"]`) will 😬 .

Given this is not a change introduced by this PR (we already have this structure on the current page, I'd be tempted to leave it as is (H1) and address it separately (it may already be in the A11y team's backlog, I suspect).

Reply:
Yeah, I spotted that [the first time I did this](https://github.com/alphagov/government-frontend/pull/1481). I think the risks of trying to cascade changes to HTML heading levels are probably too high to pick up here. I think the A11y team graded it as low priority (see my comment on the previous PR, though I can't see the Trello board any more)